etc/profile-m-z/pkglog.profile

```diff
@@ -0,0 +1,59 @@
+# Firejail profile for pklog
+# Description: Reports log of package updates
+# This file is overwritten after every install/update
+# Persistent local customizations
+include pkglog.local
+# Persistent global definitions
+include globals.local
+
+whitelist /var/log/apt/history.log
+whitelist  /var/log/dnf.rpm.log
```

Comment:
`whitelist` should go below `disable-*.inc`

```suggestion
whitelist /var/log/dnf.rpm.log
```

Reply:
Thx! Fixed it.